selectors/views/threads.test.ts

```diff
@@ -31,16 +31,23 @@ describe('selectors/views/threads', () => {
             },
             rhs: {
                 selectedPostId,
+                isSidebarOpen,
```

Comment:
shouldn't we call it `isRhsOpen` to be consistent with the naming?

Reply:
I see what you mean, It would make sense to me to name it `isOpen` since it's in `rhs` reducer. 
I guess it's called `isSidebarOpen` to differentiate from  `isMenuOpen` which is also in `rhs` reducer.